### PR TITLE
moved some logs to trace level

### PR DESCRIPTION
**Pull request by macfarla**
* move "Requesting x headers" etc to trace level logging
* use peer.shortNodeId rather than logging the whole peer record
* move "Wrote initial crypto handshake message" to trace
* also a couple of frequently logged tx pool logs

### CODE REVIEW

**Comment by macfarla** [comment]
left ChainHeadTracker at debug but use shortNodeId for peer
```
2023-11-28 12:28:12.635+10:00 | nioEventLoopGroup-3-2 | DEBUG | ChainHeadTracker | Requesting chain head info from 0xcbbaacaf47a005cb38...
2023-11-28 12:28:12.635+10:00 | nioEventLoopGroup-3-3 | DEBUG | ChainHeadTracker | Requesting chain head info from 0x4d33b9dad0673b51f9...
2023-11-28 12:28:12.635+10:00 | nioEventLoopGroup-3-1 | DEBUG | ChainHeadTracker | Requesting chain head info from 0x14ba040dd8c5b1ea60...
2023-11-28 12:28:12.635+10:00 | nioEventLoopGroup-3-6 | DEBUG | ChainHeadTracker | Requesting chain head info from 0x82b8a8b80c6c37caad...
2023-11-28 12:28:12.635+10:00 | nioEventLoopGroup-3-5 | DEBUG | ChainHeadTracker | Requesting chain head info from 0x45d51ba278b725d979...
2023-11-28 12:28:12.636+10:00 | nioEventLoopGroup-3-4 | DEBUG | ChainHeadTracker | Requesting chain head info from 0xc7d29d1bbb768e73c6...
2023-11-28 12:28:12.645+10:00 | nioEventLoopGroup-3-4 | DEBUG | WaitForPeerTask | Finished waiting for peer connection.
2023-11-28 12:28:12.645+10:00 | nioEventLoopGroup-3-2 | DEBUG | WaitForPeerTask | Finished waiting for peer connection.
2023-11-28 12:28:12.649+10:00 | nioEventLoopGroup-3-1 | DEBUG | WaitForPeerTask | Finished waiting for peer connection.
2023-11-28 12:28:12.647+10:00 | nioEventLoopGroup-3-6 | DEBUG | WaitForPeerTask | Finished waiting for peer connection.
2023-11-28 12:28:12.649+10:00 | nioEventLoopGroup-3-3 | DEBUG | WaitForPeerTask | Finished waiting for peer connection.
2023-11-28 12:28:12.648+10:00 | nioEventLoopGroup-3-5 | DEBUG | WaitForPeerTask | Finished waiting for peer connection.
2023-11-28 12:28:12.649+10:00 | nioEventLoopGroup-3-4 | INFO  | FullSyncTargetManager | Unable to find sync target. Currently checking 6 peers for usefulness
2023-11-28 12:28:12.665+10:00 | nioEventLoopGroup-3-4 | DEBUG | WaitForPeerTask | Waiting for new peer connection. 6 peers currently connected.
2023-11-28 12:28:13.120+10:00 | nioEventLoopGroup-3-2 | DEBUG | ChainHeadTracker | Retrieved chain head info 2253462 (0x71a6a92bfab342e8881834dfd8e77e359382a02ac16c5f46f8d94befec4b0241) from 0xcbbaacaf47a005cb38...
2023-11-28 12:28:13.120+10:00 | nioEventLoopGroup-3-1 | DEBUG | ChainHeadTracker | Retrieved chain head info 2253462 (0x71a6a92bfab342e8881834dfd8e77e359382a02ac16c5f46f8d94befec4b0241) from 0x14ba040dd8c5b1ea60...
2023-11-28 12:28:13.120+10:00 | nioEventLoopGroup-3-3 | DEBUG | ChainHeadTracker | Retrieved chain head info 2253462 (0x71a6a92bfab342e8881834dfd8e77e359382a02ac16c5f46f8d94befec4b0241) from 0x4d33b9dad0673b51f9...
2023-11-28 12:28:13.120+10:00 | nioEventLoopGroup-3-4 | DEBUG | ChainHeadTracker | Retrieved chain head info 2253462 (0x71a6a92bfab342e8881834dfd8e77e359382a02ac16c5f46f8d94befec4b0241) from 0xc7d29d1bbb768e73c6...
2023-11-28 12:28:13.123+10:00 | nioEventLoopGroup-3-5 | DEBUG | ChainHeadTracker | Retrieved chain head info 2253462 (0x71a6a92bfab342e8881834dfd8e77e359382a02ac16c5f46f8d94befec4b0241) from 0x45d51ba278b725d979...
2023-11-28 12:28:13.231+10:00 | nioEventLoopGroup-3-6 | DEBUG | ChainHeadTracker | Retrieved chain head info 2253462 (0x71a6a92bfab342e8881834dfd8e77e359382a02ac16c5f46f8d94befec4b0241) from 0x82b8a8b80c6c37caad...
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetPooledTransactionsFromPeerTask.java [hunk 1]
```diff
@@ -61,7 +61,11 @@ public Set<Hash> getTransactionHashes() {
   protected PendingPeerRequest sendRequest() {
     return sendRequestToPeer(
         peer -> {
-          LOG.debug("Requesting {} transaction pool entries from peer {}.", hashes.size(), peer);
+          LOG.atTrace()
+              .setMessage("Requesting {} transaction pool entries from peer {}.")
+              .addArgument(hashes.size())
+              .addArgument(peer)
```
Comment (gfukushima) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetPooledTransactionsFromPeerTask.java:67]: Do we want to use shortNodeId here too?

**Review by gfukushima — APPROVED** [review]
lgtm

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetPooledTransactionsFromPeerTask.java (hunk 1 above)
Comment (macfarla) [reply ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetPooledTransactionsFromPeerTask.java:67]: done

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/cache/TransactionLogBloomCacher.java [hunk 2]
```diff
@@ -159,7 +159,7 @@ void cacheLogsBloomForBlockHeader(
         return;
       }
       final long blockNumber = blockHeader.getNumber();
-      LOG.debug("Caching logs bloom for block {}.", "0x" + Long.toHexString(blockNumber));
+      LOG.debug("Caching logs bloom for block {}", "0x" + Long.toHexString(blockNumber));
```
Comment (fab-10) [inline ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/cache/TransactionLogBloomCacher.java:162]:
```suggestion
      LOG.atDebug().setMessage("Caching logs bloom for block {}").addArgument(() -> "0x" + Long.toHexString(blockNumber).log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java [hunk 3]
```diff
@@ -210,14 +210,14 @@ public void removeChainEstimatedHeightListener(final long listenerId) {
   }
 
   public void recordRequestTimeout(final int requestCode) {
-    LOG.debug("Timed out while waiting for response from peer {}", this.getShortNodeId());
+    LOG.debug("Timed out while waiting for response from peer {}...", this.getShortNodeId());
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java:213]:
```suggestion
    LOG.atDebug().setMessage("Timed out while waiting for response from peer {}...").addArgument(this::getShortNodeId).log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java [hunk 4]
```diff
@@ -210,14 +210,14 @@ public void removeChainEstimatedHeightListener(final long listenerId) {
...
     reputation.recordRequestTimeout(requestCode).ifPresent(this::disconnect);
   }
 
   public void recordUselessResponse(final String requestType) {
-    LOG.debug(
-        "Received useless response for request type {} from peer {}",
+    LOG.trace(
+        "Received useless response for request type {} from peer {}...",
         requestType,
         this.getShortNodeId());
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java:222]:
```suggestion
    LOG.atTrace().setMessage(
        "Received useless response for request type {} from peer {}...")
        .addArgument(requestType)
        addArgument(this::getShortNodeId).log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java [hunk 5]
```diff
@@ -125,18 +125,19 @@ protected Optional<List<BlockHeader>> processResponse(
...
-        "Received {} of {} headers requested from peer {}",
-        headersList.size(),
-        count,
-        peer.getShortNodeId());
+    LOG.atTrace()
+        .setMessage("Received {} of {} headers requested from peer {}...")
+        .addArgument(headersList.size())
+        .addArgument(count)
+        .addArgument(peer.getShortNodeId())
+        .log();
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java:133]:
```suggestion
    LOG.atTrace()
        .setMessage("Received {} of {} headers requested from peer {}...")
        .addArgument(headersList::size)
        .addArgument(count)
        .addArgument(peer::getShortNodeId)
        .log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/BufferedGetPooledTransactionsFromPeerFetcher.java [hunk 6]
```diff
@@ -89,10 +89,10 @@ public void requestTransactions() {
                 transactionTracker.markTransactionsAsSeen(peer, retrievedTransactions);
 
                 LOG.atTrace()
-                    .setMessage("Got {} transactions of {} hashes requested from peer {}")
+                    .setMessage("Got {} transactions of {} hashes requested from peer {}...")
                     .addArgument(retrievedTransactions::size)
                     .addArgument(task.getTransactionHashes()::size)
-                    .addArgument(peer)
+                    .addArgument(peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/BufferedGetPooledTransactionsFromPeerFetcher.java:95]:
```suggestion
                    .addArgument(peer::getShortNodeId)
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/BufferedGetPooledTransactionsFromPeerFetcher.java [hunk 7]
```diff
@@ -120,8 +120,8 @@ private List<Hash> getTxHashesAnnounced() {
     metrics.incrementAlreadySeenTransactions(metricLabel, alreadySeenCount);
     LOG.atTrace()
         .setMessage(
-            "Transaction hashes to request from peer {}, fresh count {}, already seen count {}")
-        .addArgument(peer)
+            "Transaction hashes to request from peer {}... fresh count {}, already seen count {}")
+        .addArgument(peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/BufferedGetPooledTransactionsFromPeerFetcher.java:124]:
```suggestion
        .addArgument(peer::getShortNodeId)
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetHeadersFromPeerByHashTask.java [hunk 8]
```diff
@@ -119,11 +119,12 @@ public static AbstractGetHeadersFromPeerTask forSingleHash(
...
-              "Requesting {} headers (hash {}...) from peer {}.",
-              count,
-              referenceHash.slice(0, 6),
-              peer.getShortNodeId());
+          LOG.atTrace()
+              .setMessage("Requesting {} headers (hash {}...) from peer {}.")
+              .addArgument(count)
+              .addArgument(referenceHash.slice(0, 6))
+              .addArgument(peer.getShortNodeId())
+              .log();
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetHeadersFromPeerByHashTask.java:127]:
```suggestion
          LOG.atTrace()
              .setMessage("Requesting {} headers (hash {}...) from peer {}.")
              .addArgument(count)
              .addArgument(() -> referenceHash.slice(0, 6))
              .addArgument(peer::getShortNodeId)
              .log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetHeadersFromPeerByNumberTask.java [hunk 9]
```diff
@@ -77,11 +77,12 @@ public static AbstractGetHeadersFromPeerTask forSingleNumber(
...
-          LOG.debug(
-              "Requesting {} headers (blockNumber {}) from peer {}.",
-              count,
-              blockNumber,
-              peer.getShortNodeId());
+          LOG.atTrace()
+              .setMessage("Requesting {} headers (blockNumber {}) from peer {}.")
+              .addArgument(count)
+              .addArgument(blockNumber)
+              .addArgument(peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetHeadersFromPeerByNumberTask.java:84]:
```suggestion
              .addArgument(peer::getShortNodeId)
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetNodeDataFromPeerTask.java [hunk 10]
```diff
@@ -66,7 +66,11 @@ public static GetNodeDataFromPeerTask forHashes(
   protected PendingPeerRequest sendRequest() {
     return sendRequestToPeer(
         peer -> {
-          LOG.debug("Requesting {} node data entries from peer {}.", hashes.size(), peer);
+          LOG.atTrace()
+              .setMessage("Requesting {} node data entries from peer {}...")
+              .addArgument(hashes.size())
+              .addArgument(peer.getShortNodeId())
+              .log();
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetNodeDataFromPeerTask.java:73]:
```suggestion
          LOG.atTrace()
              .setMessage("Requesting {} node data entries from peer {}...")
              .addArgument(hashes::size)
              .addArgument(peer::getShortNodeId)
              .log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetPooledTransactionsFromPeerTask.java [hunk 11]
```diff
@@ -61,7 +61,11 @@ public Set<Hash> getTransactionHashes() {
   protected PendingPeerRequest sendRequest() {
     return sendRequestToPeer(
         peer -> {
-          LOG.debug("Requesting {} transaction pool entries from peer {}.", hashes.size(), peer);
+          LOG.atTrace()
+              .setMessage("Requesting {} transaction pool entries from peer {}...")
+              .addArgument(hashes.size())
+              .addArgument(peer.getShortNodeId())
+              .log();
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetPooledTransactionsFromPeerTask.java:68]:
```suggestion
          LOG.atTrace()
              .setMessage("Requesting {} transaction pool entries from peer {}...")
              .addArgument(hashes::size)
              .addArgument(peer::getShortNodeId)
              .log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetReceiptsFromPeerTask.java [hunk 12]
```diff
@@ -83,7 +83,11 @@ protected PendingPeerRequest sendRequest() {
             .collect(toList());
     return sendRequestToPeer(
         peer -> {
-          LOG.debug("Requesting {} receipts from peer {}.", blockHeaders.size(), peer);
+          LOG.atTrace()
+              .setMessage("Requesting {} receipts from peer {}...")
+              .addArgument(blockHeaders.size())
+              .addArgument(peer.getShortNodeId())
+              .log();
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetReceiptsFromPeerTask.java:90]:
```suggestion
          LOG.atTrace()
              .setMessage("Requesting {} receipts from peer {}...")
              .addArgument(blockHeaders::size)
              .addArgument(peer::getShortNodeId)
              .log();
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTracker.java [hunk 13]
```diff
@@ -66,7 +66,10 @@ public static void trackChainHeadForPeers(
 
   @Override
   public void onPeerConnected(final EthPeer peer) {
-    LOG.debug("Requesting chain head info from {}", peer);
+    LOG.atDebug()
+        .setMessage("Requesting chain head info from {}...")
+        .addArgument(peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTracker.java:71]:
```suggestion
        .addArgument(peer::getShortNodeId)
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTracker.java [hunk 14]
```diff
@@ -82,14 +85,14 @@ public void onPeerConnected(final EthPeer peer) {
...
-                    .setMessage("Retrieved chain head info {} from {}")
+                    .setMessage("Retrieved chain head info {} from {}...")
                     .addArgument(
                         () ->
                             chainHeadHeader.getNumber()
                                 + " ("
                                 + chainHeadHeader.getBlockHash()
                                 + ")")
-                    .addArgument(peer)
+                    .addArgument(peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTracker.java:95]:
```suggestion
                    .addArgument(peer::getShortNodeId)
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/NewPooledTransactionHashesMessageProcessor.java [hunk 15]
```diff
@@ -86,8 +86,8 @@ private void processNewPooledTransactionHashesMessage(
 
       LOG.atTrace()
           .setMessage(
-              "Received pooled transaction hashes message from {}, incoming hashes {}, incoming list {}")
-          .addArgument(peer)
+              "Received pooled transaction hashes message from {}... incoming hashes {}, incoming list {}")
+          .addArgument(peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/NewPooledTransactionHashesMessageProcessor.java:90]:
```suggestion
          .addArgument(peer::getShortNodeId)
```

**Review by fab-10 — COMMENTED** [review]
suggested performance opt to avoid execution and memory allocation when not at trace

**Comment by macfarla** [comment]
Thanks for the review @fab-10 have actioned your comments

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/cache/TransactionLogBloomCacher.java [hunk 16]
```diff
@@ -159,7 +159,10 @@ void cacheLogsBloomForBlockHeader(
         return;
       }
       final long blockNumber = blockHeader.getNumber();
-      LOG.debug("Caching logs bloom for block {}.", "0x" + Long.toHexString(blockNumber));
+      LOG.atDebug()
+          .setMessage("Caching logs bloom for block {}")
+          .addArgument("0x" + Long.toHexString(blockNumber))
```
Comment (fab-10) [inline ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/cache/TransactionLogBloomCacher.java:164]:
```suggestion
          .addArgument(() -> "0x" + Long.toHexString(blockNumber))
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/NewPooledTransactionHashesMessageProcessor.java [hunk 17]
```diff
@@ -86,8 +86,8 @@ private void processNewPooledTransactionHashesMessage(
 
       LOG.atTrace()
           .setMessage(
-              "Received pooled transaction hashes message from {}, incoming hashes {}, incoming list {}")
-          .addArgument(peer)
+              "Received pooled transaction hashes message from {}... incoming hashes {}, incoming list {}")
+          .addArgument(peer == null ? null : peer.getShortNodeId())
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/NewPooledTransactionHashesMessageProcessor.java:90]:
```suggestion
          .addArgument(() -> peer == null ? null : peer.getShortNodeId())
```

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetHeadersFromPeerByHashTask.java [hunk 18]
```diff
@@ -119,7 +119,7 @@ public static AbstractGetHeadersFromPeerTask forSingleHash(
   protected PendingPeerRequest sendRequest() {
     return sendRequestToPeer(
         peer -> {
-          LOG.atDebug()
+          LOG.atTrace()
```
Comment (fab-10) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetHeadersFromPeerByHashTask.java:122]: can use lambda here

**Review by fab-10 — APPROVED** [review]
few more little opt, then ok

**Pull request merged**